src/common/components/externalLink/ExternalLink.tsx

```diff
@@ -4,14 +4,17 @@ import * as React from 'react';
 
 import SrOnly from '../SrOnly/SrOnly';
 
-type Props = React.AnchorHTMLAttributes<HTMLAnchorElement> & { href: string };
+type Props = React.AnchorHTMLAttributes<HTMLAnchorElement> & {
+  href: string;
+  isEmail?: boolean;
```

Comment:
For me `isEmail` is a bit specific when it comes to the API of this component. I assume that it's used here because of the `getExternalUrl` check. I would say that ideally this component does not know or care what valid `href` value it receives--I would expect that it would result in less code.

That being said, this is more of a FYI comment--this approach works for me if it's pragmatic etc.

Reply:
How would you change the API? Would you move `getExternalUrl` out of this component?

Reply:
This is our field in a Django model:
```
    external_enrolment_url = models.URLField(
        verbose_name=_("enrolment url"), blank=True, null=True
    )
```
When the field is created, it will be a basic varchar-field
```
CREATE TABLE occurrences_palvelutarjotinevent (
    -- ...
    external_enrolment_url character varying(200)
);
```
So the URL-validation is purely on ORM-level and even there it needs to be manually called. @Jokinen is right that it is a bit bad for data integrity that 1 field stores both the web-addresses and the mail-links, but I don't see much data value in this field anyway. The thing is that this is very important feature for the product owner and the deadline is only a couple of days away. 

I would create another ticket where we could investigate other options, make changes in implementation and run a data migration for the data. If we don't publish this feature now, we might not get the data at all, because the final users won't use the system or the issue would be raised as a bug ticket immediately after the launch.

Reply:
...The data migration should be easy, since only all the emails contains "@", so we could find those rows and move the dat a from a field to another.

Reply:
Refactored ExternalLink a little bit